Modules/Challenges/test/Challenges.API.Tests.Integration/Features/GetChallengeById.feature

```diff
@@ -0,0 +1,42 @@
+﻿@Integration
+Feature: Get Challenge By Id
+
+User requests a Challenge
+
+Scenario: Requesting a Challenge as an authenticated user
+	Given the user is authenticated
+	And a Challenge c
+	When a GET request is sent to the Challenges/{id} endpoint with "c.Id"
+	Then the response status code is 200
+	And the response contains a Challenge
+
+Scenario: Requesting a Challenge as an anonymous user
+	Given the user is unauthenticated
+	And a Challenge c
+	When a GET request is sent to the Challenges/{id} endpoint with "c.Id"
+	Then the response status code is 401
+
+Scenario: Requesting a nonexistent Challenge as an authenticated user
+	Given the user is authenticated
+	When a GET request is sent to the Challenges/{id} endpoint with "CHLthisisnonexisting"
+	Then the response status code is 404
+
+Scenario: Requesting a Challenge with an unsupported Accept Header as an authenticated user
+	Given the user is authenticated
+	And the Accept header is 'application/xml'
+	When a GET request is sent to the Challenges/{id} endpoint with a valid Id
+	Then the response status code is 406
+
+Scenario Outline: Requesting a Challenge with an invalid id as an authenticated user
+	Given the user is authenticated
+	When a GET request is sent to the Challenges/{id} endpoint with <id>
+	Then the response status code is 400
+	And the response content includes an error with the error code "error.platform.invalidId"
+
+Examples:
+	| id                          | description                 |
+	| CHLthishastoomanycharacters | More than 20 characters     |
+	| CHLnotenoughchars           | Less than 20 characters     |
+	| !CHLdfhuwnjdfnjnjfnd        | Contains invalid characters |
+	| PHLfdjfdjflndjkfndjk        | Does not have CHL prefix    |
+	| CHL_frfssd_fdfdsed#_        | Contains invalid characters |
```

Comment:
Did you add two lines with the same description on purpose? If so, can you put them next to each other please?